gtsfm/scene_optimizer.py

```diff
@@ -14,6 +14,7 @@
 
 from dask.delayed import Delayed
 
+import gtsfm.averaging.rotation.cycle_consistency as cycle_utils
```

Comment:
This is not a util, just import it as cycle_consistency?

Reply:
good point, updated now